### PR TITLE
SCA With Snyk

### PR DESCRIPTION
**Pull request by LivingInSyn**
This PR does a few things:

* sets up scanning of the clojure code with snyk
* sets up scanning of the base ruby code with snyk
    * part of this is committing a gemfile.lock and removing the corresponding gitignore
* removes lein nvd in favor of snyk

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,195 @@
+GIT
+  remote: https://github.com/puppetlabs/puppet.git
+  revision: 5d7a81d87fc72c7c9a48d6779ae6ec950b5fdfc9
+  ref: 6.x
+  specs:
+    puppet (6.26.0)
```
Comment (stelcodes) [inline Gemfile.lock:6]: Puppet 6.26.0 hasn't been released yet and that seems to be the cause of at least one of the failing tests

Comment (LivingInSyn) [reply Gemfile.lock:6]: the gemfile.lock was created from main with `bundle lock`

**Comment by LivingInSyn** [comment]
@austb removed the ruby scanning

**Review by austb — APPROVED** [review]
Looks good! We'll need to adjust our jenkins pipelines to remove the lein-nvd checks that run there and then this will be good to go

**Pull request merged**